etc/examples/paths.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# these can be set by folks running the examples/Makefile.testing to use a different binaries than in the path
+ECHO=${ECHO:-echo}
+JQ=${JQ:-jq}
+KUBECTL=${KUBECTL:-kubectl}
+PACHCTL=${PACHCTL:-pachctl}
+RM=${RM:-rm}
+SLEEP=${SLEEP:-sleep}
+TAR=${TAR:-tar}
+WC=${WC:-WC}
```

Comment:
Not using anything platform-specific.

Reply:
See comment below: https://github.com/pachyderm/pachyderm/pull/4715#discussion_r397365325